### PR TITLE
[Feature][QCS-1058] figaro projects with sm

### PR DESCRIPTION
**Pull request by adyagnik**

### CODE REVIEW

**Review by fabiorenesas — APPROVED** [review]
Minor comments!

File: applications/ek_ra6m4_fecs43_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml
[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_fecs43_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml:1]: Thanks Fabio, this seems to be a bug which is recently introduced. We are copying whole sensor_manager folder in the helper function to add middleware components. Hence any project we generate, also copies this file. I will create a bug ticket to fix it on sandbox and take the file off from the projects here.

[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_fecs43_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml:1]: Done. Removed this file. created https://jira.global.renesas.com/browse/QCS-1297 to fix the bug

File: applications/ek_ra6m4_fecs44_generic_uart_baremetal_serial/JLinkLog.log
[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_fecs44_generic_uart_baremetal_serial/JLinkLog.log:1]: Removed.

File: applications/ek_ra6m4_fecs44_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml
[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_fecs44_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml:1]: Removed.

File: applications/ek_ra6m4_fecs50_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml
[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_fecs50_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml:1]: Removed.

File: applications/ek_ra6m4_tgs6810_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml
[the remark this answers is not in the document]
Comment (adyagnik) [reply applications/ek_ra6m4_tgs6810_generic_uart_baremetal_serial/src/qc-middleware/sensor_manager/project_pipeline.yml:1]: Removed.

**Pull request merged**